testing/scripts/seldon_e2e_utils.py

```diff
@@ -8,6 +8,7 @@
 from subprocess import run, Popen
 import subprocess
 import json
+import yaml
```

Comment:
Would be good to add yaml into dependencies sif not already there

Reply:
It seems that below we actually use json already so may be worth sticking with that to avoid adding a new dependency (and for consistency)

Reply:
oh yes, I was thinking about it but forgot to push, will do

Reply:
Done.